spring-ai-docs/src/main/antora/modules/ROOT/pages/api/advisors.adoc

```diff
@@ -14,7 +14,7 @@ You can configure existing advisors using the xref:api/chatclient.adoc#_advisor_
 var chatClient = ChatClient.builder(chatModel)
     .defaultAdvisors(
         new MessageChatMemoryAdvisor(chatMemory), // chat-memory advisor
-        new QuestionAnswerAdvisor(vectorStore, SearchRequest.defaults()) // RAG advisor
+        new QuestionAnswerAdvisor(vectorStore, SearchRequest.builder().build()) // RAG advisor
```

Comment:
we don't even need to show `SearchRequest.builder().build()` as there is an overloaded ctor that does that anyway.